lerna.json

```diff
@@ -1,27 +1,15 @@
 {
   "lerna": "2.0.0",
+  "npmClient": "yarn",
+  "useWorkspaces": true,
   "commands": {
-    "bootstrap": {
-      "ignore": [
-        "crna-kitchen-sink",
-        "react-native-vanilla"
-      ]
-    },
     "publish": {
       "ignore": [
         "cra-kitchen-sink",
-        "crna-kitchen-sink",
```

Comment:
@Hypnosphi we don't need these because ... "private: true" in package.json?

Reply:
Because of this: https://github.com/storybooks/storybook/blob/75b0076edb15700f234d2bc1d5b3007f73095cf1/package.json#L13-L14